src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -52,13 +59,21 @@ private static UIElement CreateElement(string text, double lineHeight, TextForma
                 Background = format.BackgroundBrush,
                 Child = block,
                 CornerRadius = new CornerRadius(2),
-                Height = lineHeight - (0.25 * lineHeight),
+                Height = textView.LineHeight - (0.25 * textView.LineHeight),
                 HorizontalAlignment = HorizontalAlignment.Center,
-                Margin = new Thickness(0, -0.20 * lineHeight, 5, 0),
+                Margin = new Thickness(0, -0.20 * textView.LineHeight, 5, 0),
                 Padding = new Thickness(1),
-                VerticalAlignment = VerticalAlignment.Center,
+                SnapsToDevicePixels = textView.VisualElement.SnapsToDevicePixels,
+                UseLayoutRounding = textView.VisualElement.UseLayoutRounding,
+                VerticalAlignment = VerticalAlignment.Center
             };
 
+            // Need to set these properties to avoid unecessary reformatting because some dependancy properties
+            // affect layout
```

Comment:
Add the comment to SnapsToDevicePixels and UseLayoutRounding as well?

Reply:
Will add there as well